package/MDAnalysis/analysis/align.py

```diff
@@ -497,79 +570,38 @@ class AlignTraj(AnalysisBase):
     rmsd : Array
         Array of the rmsd values of the least rmsd between the mobile_atoms
         and reference_atoms after superposition and minimimization of rmsd
-    filename : string
+    filename : str
         String reflecting the filename of the file where mobile_atoms positions
         will be written to upon running RMSD alignment
+
+
+    Notes
+    -----
+    - If set to ``verbose=False``, it is recommended to wrap the statement in a
+      ``try ...  finally`` to guarantee restoring of the log level in the
+      case of an exception.
+    - The ``in_memory`` option changes the `mobile` universe to an
+      in-memory representation (see :mod:`MDAnalysis.coordinates.memory`)
+      for the remainder of the Python session. If ``mobile.trajectory`` is
+      already a :class:`MemoryReader` then it is *always* treated as if
+      ``in_memory`` had been set to ``True``.
+
+
+    .. versionchanged:: 0.16.0
+       new general ``weights`` kwarg replace ``mass_weights``
+
+    .. deprecated:: 0.16.0
+       Instead of ``mass_weighted=True`` use new ``weights='mass'``
+
+    .. versionchanged:: 0.17.0
+       removed deprecated `mass_weighted` keyword
+
     """
 
     def __init__(self, mobile, reference, select='all', filename=None,
-                 prefix='rmsfit_', mass_weighted=None, weights=None,
+                 prefix='rmsfit_', weights=None,
                  tol_mass=0.1, strict=False, force=True, in_memory=False,
                  **kwargs):
-        """Initialization
```

Comment:
Could you leave those here please. We semi documented this in the style guide. See https://github.com/MDAnalysis/mdanalysis/issues/919#issuecomment-236558733. I only stated there that the init parameters should be documented in `init` for abstract classes but I think this should be also true for a normal class. It has the advantage that I can easier check if a attribute is missing in the docs.